README.md

```diff
@@ -1,2 +1,42 @@
-# LeaderBoard
-Leaderboard applicaton | JavaScript
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> Leader board  application using Js/webpack
+
+
+## Built With
+
+- Js
+- web pack
```

Comment:
- You've also used HTML5 and CSS3, kindly add them also to make your readme have all the list of stack you've used in building the project.